### DISCUSSION TITLE
2024/11/30/%E5%94%90%E6%9C%9D%E7%9A%84%E7%A6%BB%E5%A9%9A%E6%96%87%E4%B9%A6

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#235] [Announcements]
# 2024/11/30/%E5%94%90%E6%9C%9D%E7%9A%84%E7%A6%BB%E5%A9%9A%E6%96%87%E4%B9%A6

我依旧每天给儿子讲述中国历史。每次，我都会提前学习一遍，然后在他睡觉前，用讲故事的方式讲给他听。

https://blog.qizhen.xyz/2024/11/30/%E5%94%90%E6%9C%9D%E7%9A%84%E7%A6%BB%E5%A9%9A%E6%96%87%E4%B9%A6

**Comment by ruanqizhen** [comment]
取其中精华，拼凑成一篇唐代离婚文书模板：

盖闻夫妇之道，乃夙世之因。累劫修持，方谐嘉耦。合卺既成，宜效松萝依倚；结缡相守，誓同日月恒常。生男则英髦秀发，咸佩簪缨；育女则淑慎柔嘉，克娴姆训。九族如春阳煦暖，六亲若明月团圆。事上恭谦有礼，怀下无私无偏。生则同衾共枕，死则聚祔合葬。

顾乃世事靡常，流光易散。琴瑟失调，参商错序；苏乳相合，尚恐异流之憾；猫鼠同窠，焉能久安之情？夫出一言，辄致妇嗔；妇微有忤，夫即加怒；伤及亲友，徒深怨怼。画堂金屋，徒悬并蒂之图；绣户绮窗，空结连理之络。若强续断弦，恐成漏舟载酒；倘犹疑不决，终致覆巢毁卵。

今既不睦，想是前世怨怼未消。既以二心不同，难归一意，会及诸亲，各迁本道。愿娘子相离之后，重梳蝉鬓，再扫娥眉，巧逞窈窕之姿，选聘高官之主。解怨释结，更莫相憎。一别两宽，各生欢喜。 

特奉三年衣粮，以表余情柔仪。

天宝二载正月庚午，某某谨立
